client/app.js

```diff
@@ -11,7 +11,28 @@ App({
     qcloud.login({
       success(res) {
         that.userInfo = res;
+        qcloud.request({
+          url: config.service.userUrl + '?OpenId=' + that.userInfo.openId,
+          method: 'GET', 
+          success: function (res) {
+            //able to get user's personal information from server
+            that.userInfo = res.body;
```

Comment:
This will overwrite the original userInfo. You probably don't want to lose the openID, avatar url, etc... I think what we should do is to add the information from response into `that.userInfo`.

Reply:
When "btnDisable" is set to true, the button will be disabled and function "infoSaveClick" won't run. When "btnDisable" is set to false, it will first store the info locally (for placeholders) and then POST. "btnDisable" will be set to true again in the first step.

Reply:
Got it!